rest_tools/server/__init__.py

```diff
@@ -21,8 +19,4 @@
     "catch_error",
     "role_authorization",
     "scope_role_auth",
-    "Auth",
-    "OpenIDAuth",
-    "Daemon",
-    "from_environment",
 ]
```

Comment:
I guess the question is, do we want to be backwards compatible and do 
```python
from ..utils.auth import Auth, OpenIDAuth
```
or should we make a minor (major?) release and break things?  If so, we could probably break a few more things. :smile:

Reply:
As it stands currently, I'm in favor of making this a minor release. Will it break things? Yes. Does it alter actual functionality? No.

Reply:
>If so, we could probably break a few more things. :smile:

What are you looking at adding?

Reply:
Not sure.  A minor release I'd probably let go as-is, whereas a major release I'd be tempted to do more.

Reply:
let's keep it minor then